tests/suites/helpers.function

```diff
@@ -23,6 +23,11 @@
 #include "mbedtls/memory_buffer_alloc.h"
 #endif
 
+#if defined(MBEDTLS_CHECK_PARAMS)
+#include <setjmp.h>
+#define MBEDTLS_PARAM_FAILED(x)    mbedtls_param_failed( #x )
+#endif
```

Comment:
I think this definition here is risky. If there is another header that include platform_util.h before this there might be a redefinition warning.